### ISSUE TITLE
How to turn on experimental JavascriptCore debugger on Android

### ISSUE THREAD

**Issue by haitaoli** [facebook/nuclide#970]
I see this new feature in changelog but couldn't figure out how to turn it on. I have 2 problems with the current debugger: 1) cannot set bp in some source files (also happens in Chrome debugger), 2) console logs are not displayed (works in Chrome). I hope the new debugger can solve at least one of them.

**Comment by johnislarry** [comment]
Here's how you can attach the JSC debugger for Android:

1. Start your app, and make sure the packager is running.
2. `cmd+shift+A` to open the debugger selection dialog
3. Select `iOS Webkit Debug Proxy`  in the `Type` dropdown (apologies for the terrible name, I'm fixing it soon)
4. Choose `Android` in the `Environment dropdown.
5. Press the `Attach` button and triggering your breakpoint should pause execution.

I will say that I'm not optimistic about `console.log` working, but that should be a simple fix that I can prioritize.  As for the breakpoint not setting... do the files you're trying to set bp's in have extensions that are not `.js`?   We used to do some filtering based on file extension.  Interesting that it also doesn't work in Chrome.

**Comment by haitaoli** [comment]
I just tried this but it doesn't pause at breakpoint, with "Remote JS debugging" on or off. Got a bunch of errors in dev console, but not sure if those are related:
<img width="936" alt="screen shot 2017-01-10 at 9 43 20 pm" src="https://cloud.githubusercontent.com/assets/280811/21837047/42db2466-d77e-11e6-87df-94ddeb5bd655.png">

The chrome debugger issue has been discussed before but nobody posted a solution: http://stackoverflow.com/questions/37078774/cant-always-set-react-native-breakpoints-in-chrome

Basically bp can only be set in the last function in the js file.

**Comment by johnislarry** [comment]
Are you on the latest version of Nuclide?  Those errors in the console are normal (unfortunately).   Will the debugger pause your app if you insert a `debugger;` statement in a codepath that you know is hit?  If so then the issue would most likely only be with BP resolution.
